-1 
Tested at: de38cb3803d1ae263c231ae65c3ba501f8c675bb
 I found an error when building: 
 
 <pre>>> Compiling  /build/cmsbuild/jenkins-workarea/workspace/ib-any-integration/CMSSW_7_4_ROOT6_X_2015-02-25-1400/src/FWCore/ROOTTests/test/write_threaded_t.cc 
>> Compiling  /build/cmsbuild/jenkins-workarea/workspace/ib-any-integration/CMSSW_7_4_ROOT6_X_2015-02-25-1400/src/FWCore/Version/src/GetFileFormatVersion.cc 
>> Compiling  /build/cmsbuild/jenkins-workarea/workspace/ib-any-integration/CMSSW_7_4_ROOT6_X_2015-02-25-1400/src/FWCore/Version/src/GetReleaseVersion.cc 
>> Compiling  /build/cmsbuild/jenkins-workarea/workspace/ib-any-integration/CMSSW_7_4_ROOT6_X_2015-02-25-1400/src/FWCore/ROOTTests/test/tformula_threaded_t.cc 
>> Building shared library tmp/slc6_amd64_gcc491/src/FWCore/Version/src/FWCoreVersion/libFWCoreVersion.so
/build/cmsbuild/jenkins-workarea/workspace/ib-any-integration/CMSSW_7_4_ROOT6_X_2015-02-25-1400/src/FWCore/ROOTTests/test/read_threaded_t.cc:14:74: fatal error: FWCore/RootAutoLibraryLoader/interface/RootAutoLibraryLoader.h: No such file or directory
 #include "FWCore/RootAutoLibraryLoader/interface/RootAutoLibraryLoader.h"
                                                                          ^
compilation terminated.
Copying tmp/slc6_amd64_gcc491/src/FWCore/Version/src/FWCoreVersion/libFWCoreVersion.so to productstore area:
Leaving library rule at FWCore/Version
</pre> 
 you can see the results of the tests here: 
 https://cmssdt.cern.ch/SDT/jenkins-artifacts/pull-request-integration/PR-7940/2926/summary.html

--- a/FWCore/ROOTTests/test/BuildFile.xml
+++ b/FWCore/ROOTTests/test/BuildFile.xml
@@ -3,7 +3,7 @@
 </bin>
 <bin file="read_threaded_t.cc" name="root_threaded_read_test">
   <use name="roothistmatrix"/>
-  <use name="FWCore/RootAutoLibraryLoader"/>
+  <use name="FWCore/FWLite"/>
 </bin>
 <bin file="tclass_methods_threaded_t.cc" name="root_threaded_tclass_methods_test">
   <use name="rootcore"/>

--- a/FWCore/ROOTTests/test/read_threaded_t.cc
+++ b/FWCore/ROOTTests/test/read_threaded_t.cc
@@ -10,8 +10,7 @@
 #include "TObjString.h"
 #include "TH1F.h"
 
-
-#include "FWCore/RootAutoLibraryLoader/interface/RootAutoLibraryLoader.h"
+#include "FWCore/FWLite/interface/AutoLibraryLoader.h"
 
 #include <memory>
 #include <cassert>
@@ -98,7 +97,7 @@ int main(int argc, char** argv) {
 
 
  
-  edm::RootAutoLibraryLoader::enable();
+  AutoLibraryLoader::enable();
 
   //Tell Root we want to be multi-threaded
   TThread::Initialize();

